src/assets/HTMLAsset.js

```diff
@@ -92,6 +93,27 @@ class HTMLAsset extends Asset {
     return newSources.join(',');
   }
 
+  collectInlineStyleDependencies(inlineStyle) {
+    // split inline styles to rule array and filter the non-epmty ones
+    const styles = inlineStyle
```

Comment:
Can we parse the css using a real css parser like postcss to process this rather than implementing our own? Check out how we collect dependencies in the CSSAsset.